boards/ambiq/apollo4p_evb/Kconfig.apollo4p_evb

```diff
@@ -4,3 +4,6 @@
 
 config BOARD_APOLLO4P_EVB
 	select SOC_APOLLO4P
+
+config SYS_CLOCK_TICKS_PER_SEC
```

Comment:
Why is this being defined at the board level?

Reply:
Odd, l have a newer local branch with it moved to soc/ambiq/Kconfig.defconfig which is where it should be.
I will definitely move this back to there.